Cargo.lock

```diff
@@ -4846,14 +4846,14 @@ dependencies = [
 
 [[package]]
 name = "jsonrpsee"
-version = "0.21.0"
+version = "0.22.4"
```

Comment:
`jsonrpsee` is getting bumped here. I would not want the dependencies in this script to conflict with the dependencies of the pop-node.

One thought is to create a `Cargo.toml` within this project directory.